src/usr/xio/xio_ev_loop.c

```diff
@@ -264,7 +264,11 @@ void *xio_ev_loop_create()
 	loop->stop_loop		= 0;
 	loop->wakeup_armed	= 0;
 	loop->deleted_events_nr = 0;
+#ifdef EPOLL_CLOEXEC
+	loop->efd		= epoll_create1(EPOLL_CLOEXEC);
+#else
 	loop->efd		= epoll_create(4096);
+#endif
```

Comment:
I don't see any point in keeping the `epoll_create(4096)`. Simply replace it with the `epoll_create1(EPOLL_CLOEXEC)`.

The `ifdef` should be removed.